aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConflictHandlerResult.java

```diff
@@ -25,17 +25,17 @@
      * Conflict is handled by discarding the local (client-side) changes, preferring whatever
      * was on the server.
      */
-    DISCARD,
+    APPLY_REMOTE,
 
     /**
      * Conflict is handled by keeping the local (client-side) changes, discarding whatever
      * was on the server.
      */
-    KEEP,
+    RETRY_LOCAL,
 
     /**
-     * Conflict is handled by return a new `Model` instance that should used
+     * Conflict is handled by return a new `Model` instance that should be used
```

Comment:
returning?

Reply:
I'm having a hard time understanding `RETRY` versus `RETRY_LOCAL`.  What version of attributes will the new model reflect?

Reply:
RETRY_LOCAL will try to update the remote data store with the remote model. While RETRY would be returned from a custom conflict handler that would allow the developer to retry with a model that may be a mix of the remote and local (if it makes sense for their use case). RETRY_LOCAL is kind of a shortcut for RETRY(localModel)

There was some discussion on RETRY when we met with the iOS team, but I think we'll need to have some solid documentation behind this one.